frontend/src/components/user/UserControl.tsx

```diff
@@ -132,8 +134,11 @@ export const UserControl = () => {
     });
 
     logoutLocalUser();
+
+    setQueryParam("auth_logout", "ok");
```

Comment:
I'm wary of adding this query param on all logouts, rather than only when we're logging out from a gated page. I can't reproduce any weird behavior based on this at the moment, but adding an unused query param just seems like bad practice and could come back to bite us later. I'm not sure immediately how we would want to implement only adding the param when logging out from gated pages, though. It may be worth some thought.